### PR TITLE
File Structure and code clean up

### PR DESCRIPTION
**Pull request by prabarai**
This PR contains

- removing inline CSS and js code
- added external CSS, JS, and media file
- added heart on hover to `yes` button
- remove two celebrate gifs
- added linear gradient animation to the background 
- changed `Sincerely, Eshan` to `Future Boyfriend, Eshan`
- changed a few codes of JS as well like  getting the element by class name instead of ID

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    btn.style.left= (Math.random()*90)+"%";
+    btn.style.top= (Math.random()*90)+"%";
+}
+function closeDialog() {
+    dialog.close();
+}
+function showConfetti(){
+    dialog.showModal();
+    document.getElementsByClassName('confetti-wrapper')[0].style.display="block";
+    document.getElementsByClassName('celebrate')[0].style.display="block";
```
Comment (prabarai) [inline script.js:15]: I changed 
    ` document.getElementById('confetti-wrapper').style.display = "block";`
to this
    `document.getElementsByClassName('celebrate')[0].style.display="block";`

File: index.html [hunk 2]
```diff
@@ -3,177 +3,57 @@
...
+        <div class="display-none celebrate">
+            <img src="https://media.tenor.com/qg8K8VOmzJwAAAAi/party-popper-confetti.gif" alt="celebrate"/>
+            <img src="https://media.tenor.com/DXL84KB6lscAAAAi/confetti-cute.gif" alt="celebrate"/>
         </div>
-        <b style="color:white;font-family: sans-serif">-Sincerely, Eshan</b>
+       <h3 class="color-white">-Future Boyfriend, Eshan</h3>
     </div>
     <!--DIALOG CONTENT-->
     <dialog id="showDialog">
-        <div style="border: 5px double #DA291C;border-radius: 4px; padding:0 24px 24px">
```
Comment (prabarai) [inline index.html:117]: glassmorphism effect on the modal is not looking good, I think there is a better way, can't think of now but will definitely change in future

Comment (prabarai) [reply index.html:117]: reverted back to white background

File: index.html [hunk 3]
```diff
@@ -3,177 +3,67 @@
...
-        dialog.close();
-    }
-    function showConfetti(){
-        dialog.showModal();
-        document.getElementById('confetti-wrapper').style.display = "block";
-        document.getElementById('celebrate').style.display = "block";
-    }
 
+        <div class="text-center">
+            <img src="https://media.tenor.com/uPqAeCeFDkkAAAAi/%E5%B9%B8%E7%A6%8F%E7%88%B1%E5%BF%83-happy.gif" alt="celebrate"/>
```
Comment (prabarai) [inline index.html:47]: #suggestion A particular gif has got empty space on the top,  need another gif

**Review by EshanB101 — COMMENTED** [review]
LGTM !! Thank for the effort, one guram for u next time

**Review by EshanB101 — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (EshanB101) [reply index.html:47]: Empty?

Comment (prabarai) [reply index.html:47]: [28d2af07-6eb6-4d90-bd4d-db1b77dd3bc1.webm](https://user-images.githubusercontent.com/94899684/210973665-fbb13f3f-52e3-4265-992d-cfd282bd3feb.webm)

Comment (EshanB101) [reply index.html:47]: Theres 2 gif here actually

image

Comment (prabarai) [reply index.html:47]: Thanks will check it out

**Pull request closed without merging**